mmseg/datasets/dataset_wrappers.py

```diff
@@ -15,10 +20,107 @@ class ConcatDataset(_ConcatDataset):
         datasets (list[:obj:`Dataset`]): A list of datasets.
     """
 
-    def __init__(self, datasets):
+    def __init__(self, datasets, separate_eval=True):
```

Comment:
Docstring for `separate_eval`.

Reply:
hi, I hava fix the issue and add unittest for it, does I need to submit a new PR or not?